With gcc 13 AVX is used in static initialization in math_test.cc
--- highway-1.1.0/CMakeLists.txt.orig	2024-03-26 06:11:37.242346683 +0100
+++ highway-1.1.0/CMakeLists.txt	2024-03-26 06:14:45.194365491 +0100
@@ -618,7 +618,6 @@ list(APPEND HWY_TEST_FILES
   hwy/contrib/image/image_test.cc
   # Disabled due to SIGILL in clang7 debug build during gtest discovery phase,
   # not reproducible locally. Still tested via bazel build.
-  hwy/contrib/math/math_test.cc
   hwy/contrib/sort/sort_test.cc
   hwy/contrib/sort/bench_sort.cc
   hwy/contrib/thread_pool/thread_pool_test.cc
